api/src/services/application.service.ts

```diff
@@ -349,6 +358,74 @@ export class ApplicationService {
     return await this.findOne(rawApplication.id, req);
   }
 
+  /*
+    this will the most recent application the user has submitted
+  */
+  async publicAppsView(
+    params: PublicAppsViewQueryParams,
+    req: ExpressRequest,
+  ): Promise<PublicAppsViewResponse> {
+    const user = mapTo(User, req['user']);
+    if (!user) {
+      throw new ForbiddenException();
+    }
```

Comment:
Just added the authorize action calls so that should be covered now! As for the second question, I wondered that too but ended up following existing patterns like the most-recent-application-query-param passes in the userId. I have no problem changing it but is there context behind that previous decision @YazeedLoonat?